pkg/comp-functions/functions/common/billing.go

```diff
@@ -75,6 +77,13 @@ func CreateBillingRecord(ctx context.Context, svc *runtime.ServiceRuntime, comp
 	}
 
 	p := &v1.PrometheusRule{
+		ObjectMeta: metav1.ObjectMeta{
+			Labels: map[string]string{
+				// This label is required, the ensure that the rule is added to Prometheus and
+				// not the Thanos Rules in the Openshift User Workload Monitoring
+				"openshift.io/prometheus-rule-evaluation-scope": "leaf-prometheus",
```

Comment:
Do I understand correctly, that this is the magic sauce that will make the billing data go to the right place in mimir?